### PR TITLE
refactor(hero): add animations to hero

### PR DESCRIPTION
**Pull request by d-kja**

### CODE REVIEW

File: src/components/magicui/box-fade-in.tsx
Comment (d-kja) [inline src/components/magicui/box-fade-in.tsx:1]: I created this one so if you wanna refactor anything or make it more extensible, just lemme know

**Comment by d-kja** [comment]
you can leave this one for last, as it might need merge with the other prs

File: src/components/shards/hero.tsx [hunk 1]
```diff
@@ -10,31 +12,43 @@ export const Hero = async () => {
 
   return (
```
Comment (d-kja) [inline src/components/shards/hero.tsx:13]: brotha, if you're planning to use theming use the goddam variables

Comment (ndelanhese) [reply src/components/shards/hero.tsx:13]: kkkkk nice cooment

**Review by ndelanhese — DISMISSED** [review]
ok

**Review by ndelanhese — APPROVED** [review]
Ok

**Pull request merged**